.circleci/config.yml

```diff
@@ -984,14 +992,18 @@ workflows:
                 - release
                 - master
                 - beta
-      - plugin-amplify_e2e_tests:
+      - hostingPROD-amplify_e2e_tests:
           filters: &ref_2
             branches:
               only:
                 - master
                 - split-test
           requires:
             - publish_to_local_registry
```

Comment:
Nit: Adding `hosting-amplify_e2e_tests` in the requires would be good, as it would run in the 4th wave of tests after running hosting.
Nit: `hostingDEV-amplify_e2e_tests` for clarity